src/cli/scaffold/rust/Cargo.template.toml

```diff
@@ -7,8 +7,9 @@
   serde = "1.0"
   serde_json = "1.0"
   serde_derive = "1.0"
-  hdk = { git = "https://github.com/holochain/holochain-rust" , branch = "develop" }
-  holochain_wasm_utils = { git = "https://github.com/holochain/holochain-rust" , branch = "develop" }
+  boolinator = "2.4"
```

Comment:
I don't think that boolinator should be included in the scaffold. It is pretty confusing

Reply:
Ok, sure yes let's remove